etc/config/analysis.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&llvm-mca
+compilerType=analysis
+supportsBinary=false
+objdumper=
+demangler=
+
+group.llvm-mca.compilers=llvm-mca
+group.llvm-mca.versionFlag=--version
```

Comment:
Note that `--version` is the default argument used if none is supplied. Also, you might want to add a `.groupName` key to better show in the UI whay thid group is

Reply:
Done.